_posts/2023-02-17-7th-workshop-announcement.md

```diff
@@ -0,0 +1,140 @@
+presentations, hands-on coding sessions, discussion/brainstorming sessions, and generally 
+be held in an informal manner (with a possible hike to a Scotish National park).**
+
+### Agenda
+
+The workshop will include:
+
+1. <b>Scientific presentations</b> by the participants (ca. 15-20 min) on topics related to the theme of the workshop. 
+These can include the state, recent developments and results of glacier modeling with OGGM.
+2. <b>Discussion/brainstorming sessions</b> focused on topics of interest, such as future developments and visions for the model.
+3. <b>Hands-on sessions</b> in smaller groups including:
+    - Beginner tutorials: these sessions are for current and future OGGM users to learn how to set-up and customize a model run.
+    - Advanced tutorials: on topics such as <i>How to contribute to OGGM</i>, <i>use OGGM with your own data</i>, or <i>use parts of OGGM with another model</i>.
+    - Q&A session: on model problems and possible solutions.
+4.  <b>Workshop excursion</b>: Ice sheets covered all of Scotland in the past, leaving an amazing landscape behind. We plan to have a one-day excursion to one of Scotland's iconic ice carved landscapes.
+
+We like to keep the schedule flexible in order to meet the needs, interests and suggestions 
+of the participants. Tentatively, we plan to have presentations mostly in the mornings and 
+group discussions and hand-on sessions in the afternoons. 
+
+### Who can participate ?
+Anyone working on glaciological modeling using OGGM,
+but also any other global glacier modeler keen on learning more about OGGM! In particular, 
+we would like to encourage model users (current and future) as well as model developers to join 
+us. The number of participants is limited to approximately 30 due to the housing limitations 
+at the University accomodation. Therefore users and developers of OGGM will be given 
+priority in case interest exceeds capacity.
+
+### Location
+The workshop will be held at the [Edinburgh Climate Change Institute (ECCI)](https://edinburghcentre.org/). ECCI is part of the School of GeoSciences, University of Edinburgh. Located in the [heart of the city,](https://goo.gl/maps/Edb3ixpE4D745icW8) is a leading centre for research, teaching, policy and practice for climate action. In general, summers in Edinburgh are reasonably warm and pleasant and the city has a lot to offer including one of the most famouse summer festivals in the world [The Fringe](https://www.edfringe.com/experience/what-is-the-festival-fringe). 
+
+### Transportation
+Edinburgh is the capital city of Scotland (in the UK) therefore, it is very accesible by flight or train. Edinburgh Airport is the closest international airport and Edinburgh Waverley is the name of the central train station. From Amsterdam (NL) there are a regular ferries to Newcastle (England), which is only approx. 2hrs by train from Edinburgh. There are also daily trains from London. 
```

Comment:
To my knowledge there are no ferries from Amsterdam to Newcastle. Some websites seem to pretend they have, but they do in the end turn out to leave from the port of IJmuiden (~25 km away from Amsterdam). Therefore I suggest changing "Amsterdam" to "The Netherlands" for now, to avoid confusion. At a later stage I could add details on how to get there.

Reply:
ah yes I'm taking one of those next week ... "The Netherlands" 😄 is indeed a more appropriate location. I shall change that

Reply:
I added the link to the ferry as well ...